src/main/java/org/webshar/hrms/dao/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package org.webshar.hrms.dao;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.webshar.hrms.models.db.Role;
+
+public interface RoleRepository extends JpaRepository<Role, Long>
+{
+}
```

Comment:
This file needs to be present under org.webshar.hrms.repository package as per naming convention practice